### PR TITLE
Dist/21 december 2022

### PR DESCRIPTION
**Pull request by luxumbra**
Latest distro done. 

Always love seeing this on this repo. 😬 
image

### CODE REVIEW

**Comment by luxumbra** [comment]
@alalonde I've fixed the conflict I had in the ledger and the numbers still look correct, so am assuming all is good. ~~Am wondering if I should also make the seed deduct part of this PR?~~ I have also done the deduct of seeds from the ledger as part of this PR.

File: scripts/seed-minting-disburse.js [hunk 1]
```diff
@@ -13,11 +13,11 @@ const NodeAddress = sc.core.address.makeAddressModule({
...
 });
 
-const MINT_TX_HASH = "https://polygonscan.com/tx/0x4e544370a25dda862278631f0f49294b8d7b387f0a49cfcf5fe5849303f26c95";
-const MINT_DATE = "Dec 5 2022";
+const MINT_TX_HASH = "https://polygonscan.com/tx/0x51a120fb7f167d4c611a6e0ca1c8105e921977efe16066190a313df3b271538d";
+const MINT_DATE = "Jan 15 2023";
 
 const LEDGER_PATH = 'data/ledger.json';
-const MINT_AMOUNTS_PATH = './scripts/toMint20Disburse.json';
+const MINT_AMOUNTS_PATH = './scripts/toMint21Disburse.json';
```
Comment (polimyl) [inline scripts/seed-minting-disburse.js:20]: dont the coordinape seeds also have to be included somehow?

Comment (alalonde) [reply scripts/seed-minting-disburse.js:20]: No, because they're not linked to SC at all

**Review by alalonde — APPROVED** [review]
lgtm

**Comment by luxumbra** [comment]
@alalonde I didn't merge right after your approval so ended up with another conflict on the ledger. Have fixed and pushed again - the conflict was due to JayFrank registering since I pushed but I've checked the order of entries and local doesnt throw any warnings so I think is good but a sanity check would be appreciated.

**Review by alalonde — COMMENTED** [review]
Thanks; ledger still looks fine

**Comment by luxumbra** [comment]
> Thanks; ledger still looks fine

Thanks.

**Pull request merged**